azure-pipelines.yml

```diff
@@ -19,7 +19,18 @@ jobs:
         displayName: 'Running $sbt +compile'
       - script: sbt +test
         displayName: 'Running $sbt +test'
-
+      - task: Bash@3
+        inputs:
+          filePath: 'script/download_spark.sh'
+          arguments: '2.4.2 2.7'
```

Comment:
do we need this argument?

Reply:
I thought if we need to support multiple versions of spark this may be useful. What do you think?

Reply:
the script doesn't support args anyway, no? (in dotnet/spark, the script downloads all the versions needed)

Reply:
Reverting back as you suggested. Will follow dotnet/spark style of script. thanks